src/pages/page-2.js

```diff
@@ -1,151 +1,127 @@
-                  {id:2, job: "Strategic Entrepreneurship @ Erasmus", description: "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.", time: "2010-heden"}],
-      qualities:  [{id:1, value: "teamplayer"},{id:2, value: "hardwerkend"},{id:3, value: "sociaal"}],
-      skills: [{id:1, value: "HTML"},{id:2, value: "CSS"},{id:3, value: "Javascript"},{id:4, value: "React"}],
-      optional: [{id:1, value: "Overige kwaliteiten"}],
-      extraList: false,
-      isHovered: false,
-    };
-    this.addList = this.addList.bind(this);
-    this.isHovered = this.isHovered.bind(this);
-    this.changeWorkExp = this.changeWorkExp.bind(this);
-    this.changeEducation = this.changeEducation.bind(this);
-    this.changeHeader = this.changeHeader.bind(this);
-  }
-
-  addList() {
-    this.setState({extraList: !this.state.extraList});
-  }
-
-  isHovered() {
-    this.setState({isHovered: !this.state.isHovered});
-    console.log("isHovered var:" + this.state.isHovered);
-  }
-
-  changeHeader(e) {
+const CVPage = ({ headerinfo, workexp, education, qualities, skills, optional, firstPage }) => {
+  const [isHovered, setIsHovered] = useState(false)
+  const [isExtraList, setIsExtraList] = useState(false)
+  const [isHeaderInfo, setIsHeaderInfo] = useState(headerinfo)
+  const [isWorkExp, setIsWorkExp] = useState(workexp)
+  const [isEducation, setIsEducation] = useState(education)
+  const [isQual, setIsQual] = useState(qualities)
+  const [isSkills, setIsSkills] = useState(skills)
+  const [isOptional, setIsOptional] = useState(optional)
+  const [isFirstPage, setIsFirstPage] = useState(firstPage)
+
+  const changeHeader = (e) => {
+      //Get values of event
+      const targetedValue = e.target.id;
+      const newValue = e.target.value;
+      const newObj = {...isHeaderInfo}
+
+      newObj[targetedValue] = newValue;
+      setIsHeaderInfo(newObj);
+    }
+
+  const changeWorkExp = (e, index) => {
     //Get values of event
     const targetedValue = e.target.id;
     const newValue = e.target.value;
-    const newObj = {...this.state.headerinfo}
-
-    newObj[targetedValue] = newValue;
-
-    this.setState({headerinfo: newObj})
-  }
-
-  changeWorkExp(e, index) {
-    //Get values of event
-    const targetedValue = e.target.id;
-    const newValue = e.target.value;
-    const newArr = [...this.state.workexp];
+    //const newArr = [...this.state.workexp];
+    const newArr = [...isWorkExp]
 
     //Change the targeted item
     newArr[index][targetedValue] = newValue;
 
     //Setting the state with changed array
-    this.setState({workexp: newArr});
+    //this.setState({workexp: newArr});
```

Comment:
Can this be removed?

Reply:
Can't find this comment in the code, I think I removed it